neuralmonkey/encoders/factored_encoder.py

```diff
@@ -82,8 +83,7 @@ def _get_birnn_cells(self):
 
     # pylint: disable=too-many-locals
     def _create_encoder_graph(self):
-        self.dropout_placeholder = tf.placeholder(tf.float32, name="dropout")
-        self.is_training = tf.placeholder(tf.bool, name="is_training")
+        self.train_mode = tf.placeholder(tf.bool, name="is_training")
```

Comment:
když už to jednotíš, tak by tohle mělo mít `name="train_mode"`.. Sice je to i tak pokaždý jinak, ale `train_mode` je ideální jméno